public/pages/DetectorConfig/containers/Features.tsx

```diff
@@ -182,17 +187,21 @@ export class Features extends Component<FeaturesProps, FeaturesState> {
 
     const featureNum = Object.keys(featureAttributes).length;
 
+    const setParamsText = `Set the index fields that you want to find anomalies for by defining
+                           the model features. You can also set other model parameters such as
+                           window size.`
+
+    const previewText = `After you set the model features and other optional parameters, you can
+                         preview your anomalies from a sample feature output.`
```

Comment:
Are these wordings reviewed by tech writer?

Also, since those are constants, you may name it like SET_PARAMS_TEXT = '....'

Reply:
The wording is based exactly on the UX mockups. I can ask the UX designer if the mockups were reviewed by a tech writer.

Sure, I will change the variable names for these constants.

Reply:
The UX designer confirmed that the mockups have been reviewed by a tech writer.